packages/client/src/client.ts

```diff
@@ -0,0 +1,119 @@
+import { networks } from "@zetachain/networks";
+import type { Wallet } from "ethers";
+import merge from "lodash/merge";
+
+import {
+  deposit,
+  getBalances,
+  getChainId,
+  getEndpoint,
+  getFees,
+  getForeignCoins,
+  getPools,
+  getSupportedChains,
+  sendZeta,
+  trackCCTX,
+  withdraw,
+} from ".";
+
+export interface ZetaChainClientParamsBase {
+  chains?: { [key: string]: any };
+  network?: string;
+}
+
+export type ZetaChainClientParams = ZetaChainClientParamsBase &
```

Comment:
Can `signer` and `wallet` really be anything?

Reply:
1ef37fa